dashboard/db/migrate/20161111020106_create_pd_enrollment_notifications.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePdEnrollmentNotifications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_enrollment_notifications do |t|
+      t.timestamps
+      t.belongs_to :pd_enrollment, null: false
+
+      # The name of notification: i.e. name of action in Pd::WorkshopMailer
+      t.string :name
```

Comment:
Note that normal workflows don't surface the `comment`. In particular, the `annotate` gem we use does not transfer the comment into the model file (based on quick experimentation).